README.md

```diff
@@ -94,6 +94,27 @@ If you want to redeploy to the same production URL you can run `yarn vercel --pr
 
 **Hint**: We recommend connecting the project GitHub repo to Vercel so you the gets automatically deployed when pushing to `main`
 
+---
+## ⚠️ Disabling type/error checks 
+> **Warning**
+> We don't encourage to use the below options
+### Disabling commit checks
+Move to `.husky/pre-commit` file and comment out `yarn lint-staged --verbose`
```

Comment:
Explain a bit what commit checks are:

"when you commit it'll automatically run checks... blabla... and it will fail if it find any type/linting error. You can disabled it by:"